src/main/java/com/customer/jwtconfig/JwtAuthenticationEntryPoint.java

```diff
@@ -0,0 +1,28 @@
+package com.customer.jwtconfig;
+
+import java.io.IOException;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.springframework.security.core.AuthenticationException;
+import org.springframework.security.web.AuthenticationEntryPoint;
+import org.springframework.stereotype.Component;
+
+import io.jsonwebtoken.ExpiredJwtException;
+import io.jsonwebtoken.MalformedJwtException;
+
+@Component
+public class JwtAuthenticationEntryPoint implements AuthenticationEntryPoint {
```

Comment:
What does this class do.. Put some comments on top of this class

Reply:
Done.